cmd/controllermanager/main.go

```diff
@@ -48,7 +48,7 @@ func main() {
 	var sciAddr string
 	flag.StringVar(&configDumpPath, "config-dump-path", "", "The filepath to dump the running config to.")
 	// TODO: Change SCI Service name to be cloud-agnostic.
-	flag.StringVar(&sciAddr, "sci-address", "gcp-manager.substratus.svc.cluster.local:10080", "The address of the Substratus Cloud Interface server.")
+	flag.StringVar(&sciAddr, "sci-address", "sci.substratus.svc.cluster.local:10080", "The address of the Substratus Cloud Interface server.")
```

Comment:
It looks like this is already on main? https://github.com/substratusai/substratus/blob/75467f6f643fa6e9b86cb4643256e21f43e1c40c/cmd/controllermanager/main.go#L51

Reply:
yes gone after rebase